config/config.yaml

```diff
@@ -118,3 +118,12 @@
         # Machine learning analysis (e.g. clustering) of the pathway output files for each dataset
         ml:
           include: true
+          # specify how many principal components to calculate
+          components: 2
+          # boolean to show the labels on the pca graph
+          labels: true
+          # 'ward', 'complete', 'average', 'single'
+          # if linkage: ward, MUST USE metric: euclidean
+          linkage: 'ward'
+          # 'euclidean', 'l1', 'l2', 'manhattan', 'cosine'
+          metric: 'euclidean'
```

Comment:
The two clustering frameworks support different linkage and metric arguments. We decided to support only the options that work with both frameworks.